### ISSUE TITLE
ufbx_mesh_material::face_indices::count wrong value

### ISSUE THREAD

**Issue by Trider12** [bqqbarbhg/ufbx#44]
The comments in the header say that:
```cpp
// Indices to `ufbx_mesh.faces[]` that use this material.
// Always contains `num_faces` elements.
ufbx_uint32_list face_indices; 
```
So I expect `face_indices::count` to be equal to `num_faces`. But in the inspector I see the following:

![image](https://user-images.githubusercontent.com/47952308/177124313-e4654395-aaa5-4f3d-ab6f-054c623d8751.png)

And because of that `face_indices[i]` triggers an assert `ufbx_assert(index < count);`, but `face_indices.data[i]` is fine.

**Comment by bqqbarbhg** [comment]
Thanks for the issue! Ah yes I just hit this one too in a feature branch, I'll merge a hotfix to this as soon as tests pass. 

By the way if you're using the MSVC debugger I recommend downloading [misc/ufbx.natvis](https://github.com/bqqbarbhg/ufbx/blob/master/misc/ufbx.natvis) if possible, you can either add it to your project or `%USERPROFILE%\Documents\Visual Studio $Version\Visualizers`, that way you can expand list contents easily.

**Comment by bqqbarbhg** [comment]
Should be fixed now in `master` (ee0382cd)

**Comment by Trider12** [comment]
@bqqbarbhg, thanks for the tip, but VS2019 does not support using `.natvis` files with CMake projects.

I confirm that the issue is fixed.
